BitmarkRegistry/Controllers/Main/Properties/RegisterPropertyViewController.swift

```diff
@@ -130,25 +130,32 @@ extension RegisterPropertyViewController: UIImagePickerControllerDelegate, UINav
 // MARK: - UIDocumentPickerDelegate
 extension RegisterPropertyViewController: UIDocumentPickerDelegate {
   func documentPicker(_ controller: UIDocumentPickerViewController, didPickDocumentAt url: URL) {
-    let didStartAccessing = url.startAccessingSecurityScopedResource()
-    defer {
-      if didStartAccessing {
-        url.stopAccessingSecurityScopedResource()
+    activityIndicator.startAnimating()
+    disabledScreen.isHidden = false
+    UIApplication.shared.beginIgnoringInteractionEvents()
+
+    DispatchQueue.main.async { [weak self] in
```

Comment:
This func is on mainthread already.

Reply:
Yes, It's on mainthread. But I call with async, so it should move to background thread and not block the mainthread executation.